action.yaml

```diff
@@ -59,6 +59,14 @@ inputs:
     description: Ignore unfixed vulnerabilities (default "false")
     required: false
     default: "false"
+  trivy-db-repository:
+    description: OCI repository(ies) to retrieve trivy-db in order of priority
+    required: false
+    default: "ghcr.io/aquasecurity/trivy-db:2,public.ecr.aws/aquasecurity/trivy-db:2"
+  trivy-java-db-repository:
+    description: OCI repository(ies) to retrieve trivy-java-db in order of priority
+    required: false
+    default: "ghcr.io/aquasecurity/trivy-java-db:1,public.ecr.aws/aquasecurity/trivy-db:1"
```

Comment:
Any reasons to specify container tags here?

Reply:
Because default value contains image tag.

```
$ trivy image -h | grep db-repository
      --db-repository strings        OCI repository(ies) to retrieve trivy-db in order of priority (default [ghcr.io/aquasecurity/trivy-db:2])
      --java-db-repository strings   OCI repository(ies) to retrieve trivy-java-db in order of priority (default [ghcr.io/aquasecurity/trivy-java-db:1])
```

Reply:
I think default values can change in future updates. Are we planning to keep updating tag values accordingly?

Reply:
Trivy plans to implements registry fallbacks. If it will be completed, I will revert this PR.